deltachat-contact-tools/src/lib.rs

```diff
@@ -44,7 +44,8 @@ use regex::Regex;
 pub struct VcardContact {
     /// The email address, vcard property `email`
     pub addr: String,
```

Comment:
if it is an vcard from the phone's address book, and doesn't have addr, it is just sharing phone number and other data, should it still have vcard view-type or shouldn't it rather be file viewtype since it is not relevant to DC, it is not a DC contact what is being shared :thinking: how is this handled?

Reply:
Good that you bring that up! I think it should simply be a normal attached file, otherwise we have to tell all UIs that they need to handle this case. This was surprisingly easy to implement: f95273491

Reply:
great!